bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java

```diff
@@ -344,9 +345,10 @@ public static Resource getWrappedImageResourceWithInheritance(Resource resource,
         String fileReference = properties.get(DownloadResource.PN_REFERENCE, String.class);
         Resource fileResource = resource.getChild(DownloadResource.NN_FILE);
         boolean imageFromPageImage = properties.get(PN_IMAGE_FROM_PAGE_IMAGE, StringUtils.isEmpty(fileReference) && fileResource == null);
+        boolean disblePageImageInheritance = currentStyle != null ? currentStyle.get(Image.PN_PAGE_IMAGE_INHERITANCE_DISABLED, false)  : false;
```

Comment:
Small typo in variable name, I believe it should be dis**a**blePageImageInheritance